app/templates/components/forms/admin/content/social-links-form.hbs

```diff
@@ -1,27 +1,27 @@
-<form class="ui form" {{action 'submit' on='submit'}}>
+<form class="ui form {{if isLoading 'loading'}}" {{action 'submit' on='submit'}} novalidate>
   <div class="field">
     <label>{{t 'Support'}}</label>
-    {{input type='url' name='support'}}
+    {{input type='url' name='support' value=socials.supportUrl}}
   </div>
   <div class="field">
     <label>{{t 'Twitter'}}</label>
-    {{input type='url' name='twitter'}}
+    {{input type='url' name='twitter' value=socials.twitterUrl}}
   </div>
   <div class="field">
     <label>{{t 'Facebook'}}</label>
-    {{input type='url' name='facebook'}}
+    {{input type='url' name='facebook' value=socials.facebookUrl}}
```

Comment:
@niranjan94 We will need to modify the component. Please consider the following case

![screenshot from 2017-07-23 18-52-41](https://user-images.githubusercontent.com/12807846/28499649-28038370-6fd8-11e7-8ea3-4c4b3b3e5dc3.png)

Also the component produces a segmentedLink which is an object containing protocol & the link. 

Please guide, Thanks.